archinstall/lib/disk/partition.py

```diff
@@ -186,7 +186,7 @@ def _call_lsblk(self) -> Dict[str, Any]:
 			try:
 				output = SysCommand(f"lsblk --json -b -o+LOG-SEC,SIZE,PTTYPE,PARTUUID,UUID,FSTYPE {self.device_path}").decode('UTF-8')
 			except SysCallError as error:
-				# It appears as if lsblk can return exit codes like 8192 to indicate something.
+				# It appears as if lsblk can return exit codes like 32 to indicate something.
```

Comment:
The exit status of 32 is "none of specified devices found". [1] Should this line and maybe even the following line then be deleted?

[1] https://man.archlinux.org/man/lsblk.8#EXIT_STATUS